commands/proxy/proxy-set.ts

```diff
@@ -103,6 +103,8 @@ export class ProxySetCommand extends ProxyCommandBase {
 			this.$logger.warn(`Note that storing credentials is not supported on ${platform()} yet.`);
 		}
 
+		this.$logger.warn("Note that 'npm' and 'Gradle' need to be configured separately to work with a proxy. Run 'tns proxy set --help' for more information.");
```

Comment:
As this code is shared between AppBuilder and NativeScript CLI, you should use `$staticConfig.CLIENT_NAME` instead of `tns` in the message.
Also `Gradle`  is not used in AppBuilder CLI.
Maybe we can place some if here to print the message only when we are in NativeScript CLI scenario.